File: SchoolManagementApi/DTOs/Enrollments/EnrollmentResponseDto.cs
namespace SchoolManagementApi.DTOs.Enrollments;

public class EnrollmentResponseDto
{
    public int Id { get; set; }
    public int StudentId { get; set; }
    public string StudentName { get; set; } = null!;
    public int SubjectId { get; set; }
    public string SubjectName { get; set; } = null!;
    public DateTime CreatedAt { get; set; }
}

File: SchoolManagementApiNode/src/enrollments/dto/enrollment-response.dto.ts
import { ApiProperty } from '@nestjs/swagger';

export class EnrollmentResponseDto {
  @ApiProperty()
  id: number;

  @ApiProperty()
  studentId: number;

  @ApiProperty()
  studentName: string;

  @ApiProperty()
  subjectId: number;

  @ApiProperty()
  subjectName: string;

  @ApiProperty()
  createdAt: Date;
}
